chromebook-setup.sh

```diff
@@ -142,18 +159,29 @@ or to do the same to use NFS for the root filesystem:
     exit $arg_ret
 }
 
-opts=$(getopt -o "s:" -l "storage:,architecture:" -- "$@")
+opts=$(getopt -o "h,s:" -l "help,distro:,repository:,storage:,architecture:" -- "$@")
 eval set -- "$opts"
 
 while true; do
     case "$1" in
+	--help|-h)
+	    print_usage_exit
+	    ;;
+	--distro)
+	    CB_DISTRO="$2"
+	    shift 2
+	    ;;
+	--repository)
```

Comment:
Should we use --kernel instead of --repository? Repository looks a bit ambiguous to me. What do you think?

Reply:
I actually called this kernel at the beginning but then decided to go with repo. I'll change it.